save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -158,9 +158,10 @@ class OrganizationService(
      * @throws NoSuchElementException
      */
     fun saveAvatar(name: String, relativePath: String) {
-        val organization = organizationRepository.findByName(name)?.apply {
-            avatar = relativePath
-        } ?: throw NoSuchElementException("Organization with name [$name] was not found.")
+        val organization = organizationRepository.findByNameAndStatusIn(name, setOf(OrganizationStatus.CREATED, OrganizationStatus.DELETED))
+            ?.apply {
+                avatar = relativePath
+            } ?: throw NoSuchElementException("Organization with name [$name] was not found.")
```

Comment:
we have `orNotFound `:
```suggestion
            }.orNotFound { "Organization with name [$name] was not found." }
```

Reply:
Done